[2022-08-13 11:28:15,869] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-13T11:28:13.109921+00:00 [queued]>
[2022-08-13 11:28:15,885] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-13T11:28:13.109921+00:00 [queued]>
[2022-08-13 11:28:15,885] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 11:28:15,886] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-13 11:28:15,887] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 11:28:15,905] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): download_raw_data> on 2022-08-13 11:28:13.109921+00:00
[2022-08-13 11:28:15,910] {standard_task_runner.py:52} INFO - Started process 419 to run task
[2022-08-13 11:28:15,914] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'download_raw_data', 'manual__2022-08-13T11:28:13.109921+00:00', '--job-id', '300', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpvtvi208o', '--error-file', '/tmp/tmpg6a202wr']
[2022-08-13 11:28:15,915] {standard_task_runner.py:80} INFO - Job 300: Subtask download_raw_data
[2022-08-13 11:28:15,984] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-13T11:28:13.109921+00:00 [running]> on host 4bbb34367c41
[2022-08-13 11:28:16,083] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=download_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-13T11:28:13.109921+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-13T11:28:13.109921+00:00
[2022-08-13 11:28:16,085] {ETL_Raizen.py:40} INFO - Downloading .xls raw data
[2022-08-13 11:28:19,645] {ETL_Raizen.py:42} INFO - Succesfully downloaded .xls raw data
[2022-08-13 11:28:19,646] {python.py:173} INFO - Done. Returned value was: None
[2022-08-13 11:28:19,677] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=download_raw_data, execution_date=20220813T112813, start_date=20220813T112815, end_date=20220813T112819
[2022-08-13 11:28:19,709] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-13 11:28:19,761] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
